docs/misp2_installation_manual_18.04.md

```diff
@@ -519,6 +592,15 @@ The steps for configuring HTTPS:
 
 #### 5.4.1 Service parameters
 
+Before moving forward with the configuration, create a `p12` trust store
+containing the correct certificate based on the documentation provided in [SK-s
+JAVA client source
+repository](https://github.com/SK-EID/mid-rest-java-client#how-to-obtain-server-certificate).
+
+Once the trust store is created, move it to the
+`/var/lib/tomcat8/webapps/misp2/WEB-INF/classes` folder and update the file
+permissions so that it is accessible by the system user `tomcat8`.
```

Comment:
It would be good to specify the command that's required to set the correct permissions.

Reply:
Added example of chown.